.storybook/main.ts

```diff
@@ -23,15 +23,15 @@ const config: StorybookConfig = {
   },
 
   refs: {
-    flowbite: {
-      title: "Flowbite",
-      url: "https://storybook.flowbite-react.com/",
-      expanded: false, // Optional, true by default
-    },
+    // flowbite: {
```

Comment:
Intentionally commented out?

Reply:
Aye, it was getting annoying to me. I might restore it one day on demand as I work.